### DISCUSSION TITLE
redhat IDE

### DISCUSSION THREAD

**Discussion by VasanthaRoyal** [castorini/anserini#2166] [Q&A]
I am using redhat IDE and java 8 version but  iam fasing this issue can any one help me on this.
 Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project external-properties-demo: Fatal error compiling: invalid flag: --release -> [Help 1]

**Comment by lintool** [comment]
Hi @VasanthaRoyal - unfortunately, I don't think anyone on the team has experience with the redhat IDE
